### PR TITLE
Update data processor and Solr schema

### PR DESCRIPTION
**Pull request by jswk**
Update queries to v2.

Create configset oa_prod_20211208_v2.

### CODE REVIEW

File: transform/v2/tsv-to-jsonl.py [hunk 1]
```diff
@@ -0,0 +1,121 @@
...
+
+# @pytest.mark.parametrize("s,expected", [
+#     ("", True),
+#     ("   ", True),
+#     ("\\N", True),
+#     ("  \\N ", True),
+#     ("  foo ", False),
+# ])
+# def test_empty_or_n(s: str, expected: bool):
+#     assert empty_or_n(s) is expected
```
Comment (danielkryska) [inline transform/v2/tsv-to-jsonl.py:121]: Are those comments necessary?

File: transform/v2/tsv-to-jsonl.py [hunk 2]
```diff
@@ -0,0 +1,121 @@
+import fileinput
+import csv
+import os
+import sys
+import json
+
+
+# import pytest
+
```
Comment (danielkryska) [inline transform/v2/tsv-to-jsonl.py:9]: Are those comments necessary?

File: transform/v2/tsv-to-jsonl.py [hunk 3]
```diff
@@ -0,0 +1,121 @@
+import fileinput
```
Comment (danielkryska) [inline transform/v2/tsv-to-jsonl.py:1]: Have you written the module/class or its copy-paste solution?

File: solr/config/configsets/oa_prod_20211208_v2/conf/lang/stemdict_nl.txt [hunk 4]
```diff
@@ -0,0 +1,6 @@
+# Set of overrides for the dutch stemmer
+# TODO: load this as a resource from the analyzer and sync it in build.xml
```
Comment (danielkryska) [inline solr/config/configsets/oa_prod_20211208_v2/conf/lang/stemdict_nl.txt:2]: Please move the TODO to the repository as the ticket. It'll be better to track.

File: solr/config/configsets/oa_prod_20211208_v2/conf/lang/hyphenations_ga.txt [hunk 5]
```diff
@@ -0,0 +1,5 @@
+# Set of Irish hyphenations for StopFilter
+# TODO: load this as a resource from the analyzer and sync it in build.xml
```
Comment (danielkryska) [inline solr/config/configsets/oa_prod_20211208_v2/conf/lang/hyphenations_ga.txt:2]: Please move the TODO to the repository as the ticket. It'll be better to track.

File: solr/config/configsets/oa_prod_20211208_v2/conf/lang/contractions_it.txt [hunk 6]
```diff
@@ -0,0 +1,23 @@
+# Set of Italian contractions for ElisionFilter
+# TODO: load this as a resource from the analyzer and sync it in build.xml
```
Comment (danielkryska) [inline solr/config/configsets/oa_prod_20211208_v2/conf/lang/contractions_it.txt:2]: Please move the TODO to the repository as the ticket. It'll be better to track.

File: solr/config/configsets/oa_prod_20211208_v2/conf/lang/contractions_ga.txt [hunk 7]
```diff
@@ -0,0 +1,5 @@
+# Set of Irish contractions for ElisionFilter
+# TODO: load this as a resource from the analyzer and sync it in build.xml
```
Comment (danielkryska) [inline solr/config/configsets/oa_prod_20211208_v2/conf/lang/contractions_ga.txt:2]: Please move the TODO to the repository as the ticket. It'll be better to track.

File: solr/config/configsets/oa_prod_20211208_v2/conf/lang/contractions_fr.txt [hunk 8]
```diff
@@ -0,0 +1,15 @@
+# Set of French contractions for ElisionFilter
+# TODO: load this as a resource from the analyzer and sync it in build.xml
```
Comment (danielkryska) [inline solr/config/configsets/oa_prod_20211208_v2/conf/lang/contractions_fr.txt:2]: Please move the TODO to the repository as the ticket. It'll be better to track.

File: solr/config/configsets/oa_prod_20211208_v2/conf/lang/contractions_ca.txt [hunk 9]
```diff
@@ -0,0 +1,8 @@
+# Set of Catalan contractions for ElisionFilter
+# TODO: load this as a resource from the analyzer and sync it in build.xml
```
Comment (danielkryska) [inline solr/config/configsets/oa_prod_20211208_v2/conf/lang/contractions_ca.txt:2]: Please move the TODO to the repository as the ticket. It'll be better to track.

File: README.md [hunk 10]
```diff
@@ -48,6 +54,25 @@ docker run --rm -v "$PWD/tmp/000017_0.jsonl:/mydata/data.jsonl" \
            post -c ess /mydata/data.jsonl
 ```
 
+### queries v2
+
```
Comment (danielkryska) [inline README.md:58]: What feature provides `queries v2`? Could you add a short description? I've been in at the Confluence page, but there are only SQL queries, without any context.

File: README.md [hunk 11]
```diff
@@ -48,6 +54,25 @@ docker run --rm -v "$PWD/tmp/000017_0.jsonl:/mydata/data.jsonl" \
...
+https://docs.cyfronet.pl/display/FID/Queries+v2
+
+Either use directly the jsonl file or transform the original using (assuming the file is placed in `tmp/qv2-pub/000550_0`):
+```
+python transform/v2/tsv-to-jsonl.py tmp/qv2-pub/000550_0 > tmp/qv2-pub/000550_0.jsonl
+```
+
+To process datafiles without journal set envvar `OMIT_JOURNAL=1` for processing.
```
Comment (danielkryska) [inline README.md:66]: Maybe we should create some dictionary of concepts. For example what it's the `journal`?

**Review by danielkryska — APPROVED** [review]

File: transform/v2/tsv-to-jsonl.py (hunk 3 above)
Comment (jswk) [reply transform/v2/tsv-to-jsonl.py:1]: It's copied and then modified to match the data we get.

File: transform/v2/tsv-to-jsonl.py (hunk 1 above)
Comment (jswk) [reply transform/v2/tsv-to-jsonl.py:121]: Necessary for now, as I use a bare python for processing and it doesn't have pytest installed then.
Nevertheless, I'll extract it to a separate file.

File: solr/config/configsets/oa_prod_20211208_v2/conf/lang/stemdict_nl.txt (hunk 4 above)
Comment (jswk) [reply solr/config/configsets/oa_prod_20211208_v2/conf/lang/stemdict_nl.txt:2]: Just copied from the default schema, disregard these.

File: README.md (hunk 11 above)
Comment (jswk) [reply README.md:66]: Maybe, but maybe it should be kept on confluence? I don't know yet.

File: README.md (hunk 10 above)
Comment (jswk) [reply README.md:58]: Yup, Hive queries. Their output is transformed by this script.

**Pull request merged**